source/package.json

```diff
@@ -22,6 +22,7 @@
     "typescript": "^4.9.4"
   },
   "dependencies": {
+    "@aws-cdk/aws-servicecatalogappregistry-alpha": "^2.39.1-alpha.0",
```

Comment:
Match this with your cdk version. Though you will have to upgrade to 2.61 because I think 2.59 is undeployable because of this bug: https://github.com/aws/aws-cdk/issues/23641